{{cookiecutter.project_slug}}/config/websocket.py

```diff
@@ -0,0 +1,13 @@
+async def websocket_application(scope, receive, send):
```

Comment:
Should this be defined outside of the `config`, in one of the apps maybe? It sounds like it's more part of the application than the config...

Reply:
I was debating on that, and yes, it's probably best if it's in the users app.

Reply:
I looked back at my previous code, and I'm not sure if I should (or can) import a little bit of what I've done. The gist of my dilemma is that I created a custom async_urlpatterns as a dict instead of a list with their values as dicts that represent each websocket type, i.e. connect, disconnect, and receive. This value dicts have functions that take the same scope, receive, and send parameters. That's probably the reason why I initially had `websocket.py` in config instead of users app.

However, that's websocket pioneering which probably isn't meant to be in this repo; on the other hand, I don't like having websocket_application in the users app simply because people will think a client (whether or not malicious) can connect to any websocket URL.

So I suppose I'm just going to leave it in config for now...